<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schriftfeld</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="butAbbruch">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>618</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbruch</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>90</y>
     <width>88</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Gemarkung</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt1">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>315</y>
     <width>106</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(221, 221, 221)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labActLayer">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>265</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Projekt-Grunddaten</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>33</y>
     <width>289</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="butOK">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>618</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>309</y>
     <width>481</width>
     <height>283</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Icons/Schriftfeld.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>315</y>
     <width>334</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>342</y>
     <width>364</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>369</y>
     <width>238</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt5">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>369</y>
     <width>232</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt6">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>393</y>
     <width>469</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(221, 221, 221)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt7">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>432</y>
     <width>73</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(221, 221, 221)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt8">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>432</y>
     <width>100</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt9">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>432</y>
     <width>91</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt10">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>432</y>
     <width>208</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(221, 221, 221)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt11">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>468</y>
     <width>127</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt12">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>468</y>
     <width>229</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt13">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>468</y>
     <width>115</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt14">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>507</y>
     <width>229</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(221, 221, 221)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt15">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>507</y>
     <width>115</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(221, 221, 221)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSchnittNr_16">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>507</y>
     <width>127</width>
     <height>40</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt16">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>546</y>
     <width>67</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(221, 221, 221)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt17">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>546</y>
     <width>94</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(221, 221, 221)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt18">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>546</y>
     <width>85</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(221, 221, 221)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>435</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Maßstab</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>432</y>
     <width>82</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lagebezug</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>432</y>
     <width>82</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Höhenbezug</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>432</y>
     <width>82</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dateiname</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>477</y>
     <width>124</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Geländeaufnahme</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLayProjektname">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>364</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>63</y>
     <width>88</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Projektname</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLayGemarkung">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>87</y>
     <width>364</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>117</y>
     <width>88</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Gemeinde</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLayGemeinde">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>114</y>
     <width>364</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>144</y>
     <width>88</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Kreis</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>198</y>
     <width>88</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Höhenbezug</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>171</y>
     <width>88</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Lagebezug</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>507</y>
     <width>124</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nachbearbeitung/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>525</y>
     <width>124</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Digitalisierung</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>468</y>
     <width>64</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Datum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>507</y>
     <width>64</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Datum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>546</y>
     <width>64</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kontrolle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>546</y>
     <width>64</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>546</y>
     <width>64</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Datum</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLayGelaendeaufnahme">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>222</y>
     <width>364</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>225</y>
     <width>103</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Geländeaufname</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>312</y>
     <width>475</width>
     <height>277</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cboLayLagebezug">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>168</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>RD83/GK5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RD83/GK4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UTM33</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cboLayHoehenbezug">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>195</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>DHHN 2016</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cboLayKreis">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>141</y>
     <width>364</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Bautzen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chemnitz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dresden</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Erzgebirkskreis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Görlitz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Leipzig Stadt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Leipzig</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Meißen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mittelsachsen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nordsachsen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sächsische Schweiz-Osterzgebirge</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vogtlandgreis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Zwickau</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txtLayGelaendeaufnahmeDatum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>249</y>
     <width>364</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>258</y>
     <width>103</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Geländeaufname</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>246</y>
     <width>103</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Datum</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>594</y>
     <width>475</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Felder in grau werden im Layout manuell oder automatisch ausgefüllt.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt19">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>315</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(221, 221, 221)</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>butAbbruch</zorder>
  <zorder>label_4</zorder>
  <zorder>txt1</zorder>
  <zorder>labActLayer</zorder>
  <zorder>line</zorder>
  <zorder>butOK</zorder>
  <zorder>txt3</zorder>
  <zorder>txt2</zorder>
  <zorder>txt4</zorder>
  <zorder>txt5</zorder>
  <zorder>txt6</zorder>
  <zorder>txt7</zorder>
  <zorder>txt8</zorder>
  <zorder>txt9</zorder>
  <zorder>txt10</zorder>
  <zorder>txt11</zorder>
  <zorder>txt12</zorder>
  <zorder>txt13</zorder>
  <zorder>txt14</zorder>
  <zorder>txt15</zorder>
  <zorder>txtSchnittNr_16</zorder>
  <zorder>txt16</zorder>
  <zorder>txt17</zorder>
  <zorder>txt18</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>txtLayProjektname</zorder>
  <zorder>label_9</zorder>
  <zorder>txtLayGemarkung</zorder>
  <zorder>label_10</zorder>
  <zorder>txtLayGemeinde</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_8</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>txtLayGelaendeaufnahme</zorder>
  <zorder>label_20</zorder>
  <zorder>groupBox</zorder>
  <zorder>cboLayLagebezug</zorder>
  <zorder>cboLayHoehenbezug</zorder>
  <zorder>cboLayKreis</zorder>
  <zorder>txtLayGelaendeaufnahmeDatum</zorder>
  <zorder>label_21</zorder>
  <zorder>label_22</zorder>
  <zorder>label_23</zorder>
  <zorder>txt19</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
